python/ray/data/_internal/arrow_block.py

```diff
@@ -452,7 +447,7 @@ def sort_and_partition(
         partitions.append(table.slice(last_idx))
         return partitions
 
-    def combine(self, key: str, aggs: Tuple[AggregateFn]) -> Block:
+    def combine(self, key: str, aggs: Tuple["AggregateFn"]) -> Block:
```

Comment:
Avoids circular dependency singe aggregate class uses` SortKey` for key validation now.